### ISSUE TITLE
Compiled API documentation

### ISSUE THREAD

**Issue by danielk333** [MITHaystack/digital_rf#37]
Most of the python and c code has well structured in-code documentation but its not compiled anywhere, at least where I can easily find it. This mean one has to open up the library source code to look up the API-description. Maybe a simple sphinx/doxygen run could generate the HTML and github-pages could serve them?

Just a suggestion for enhancement :)

**Comment by ryanvolz** [comment]
Yeah, I definitely want to get sphinx going and upload the result to Read the Docs.
